### PR TITLE
Sept 2024 releases announcement blog post

### PR DESCRIPTION
**Pull request by ForgottenProgramme**
Announcing the 24.9.0 releases of conda, conda-build and conda-libmamba-solver

### CODE REVIEW

File: static/img/blog/24-09-30-conda-releases/banner.png
Comment (kenodegard) [inline static/img/blog/24-09-30-conda-releases/banner.png:1]: This folder should be same name as the blogpost: `2024-09-30-...`

File: blog/2024-09-30-conda-releases.mdx
Comment (kenodegard) [inline blog/2024-09-30-conda-releases.mdx:1]: We're missing the special announcement

Comment (kenodegard) [inline blog/2024-09-30-conda-releases.mdx:1]: Let's rename this to match the `slug`

File: static/img/blog/24-09-30-conda-releases/banner.png
Comment (kenodegard) [reply static/img/blog/24-09-30-conda-releases/banner.png:1]: If we rename the .mdx to match the `slug` (https://github.com/conda/conda-dot-org/pull/212/files#r1783086188) we should make the same change here

**Comment by ForgottenProgramme** [comment]
superseded by #213

**Pull request closed without merging**